Formula/swagger-codegen@2.rb

```diff
@@ -0,0 +1,38 @@
+class SwaggerCodegenAT2 < Formula
+  desc "Generate clients, server stubs, and docs from an OpenAPI spec"
+  homepage "https://swagger.io/swagger-codegen/"
+  url "https://github.com/swagger-api/swagger-codegen/archive/v2.4.0.tar.gz"
+  sha256 "76940646d8cb9e65876a6c0829361ff99ded97126b97d9e954eca03682b287aa"
+
+  keg_only :versioned_formula
+
+  depends_on "maven" => :build
+  depends_on :java => "1.8"
+
+  def install
+    # Need to set JAVA_HOME manually since maven overrides 1.8 with 1.7+
+    cmd = Language::Java.java_home_cmd("1.8")
+    ENV["JAVA_HOME"] = Utils.popen_read(cmd).chomp
+
+    system "mvn", "clean", "package"
+    libexec.install "modules/swagger-codegen-cli/target/swagger-codegen-cli.jar"
+    bin.write_jar_script libexec/"swagger-codegen-cli.jar", "swagger-codegen"
+  end
+
+  test do
+    (testpath/"minimal.yaml").write <<~EOS
+      ---
+      swagger: '2.0'
+      info:
+        version: 0.0.0
+        title: Simple API
+      paths:
+        /:
+          get:
+            responses:
+              200:
+                description: OK
+    EOS
+    system "#{bin}/swagger-codegen", "generate", "-i", "minimal.yaml", "-l", "swagger"
```

Comment:
Please add some assertions of output/generated files.

Reply:
Thanks :) I just added an assertion for the output